wagtail/admin/static_src/wagtailadmin/scss/layouts/workflow-edit.scss

```diff
@@ -0,0 +1,31 @@
+@import '../../../../../../client/scss/tools/mixins.general';
+@import 'page-editor.scss';
```

Comment:
stylelint fails with:

```
wagtail/admin/static_src/wagtailadmin/scss/layouts/workflow-edit.scss
  2:22  ✖  Unexpected extension       scss/at-import-partial-extension-blacklist
                 ".scss" in imported partial name
 ```

so drop `.scss`? also, would it not make sense to include `page-editor.scss` directly in the template? that way it is more explicit

Reply:
Should already be on the template through `{% include "wagtailadmin/pages/_editor_css.html" %}`

This import is required in order to get the `$object-title-height` variable.

Reply:
hmm, this includes so much extra just for one var. why not move that to `_variables.scss`?

Reply:
Done! Wonder why it wasn't there in the first place